pkg/ui/workspaces/cluster-ui/src/api/databasesApi.ts

```diff
@@ -0,0 +1,56 @@
+// Copyright 2022 The Cockroach Authors.
+//
+// Use of this software is governed by the Business Source License
+// included in the file licenses/BSL.txt.
+//
+// As of the Change Date specified in that file, in accordance with
+// the Business Source License, use of this software will be governed
+// by the Apache License, Version 2.0, included in the file
+// licenses/APL.txt.
+
+import {
+  executeInternalSql,
+  SqlExecutionRequest,
+  sqlResultsAreEmpty,
+} from "./sqlApi";
+import { PROMISE_TIMEOUT, withTimeout } from "./util";
+import moment from "moment";
+
+export type DatabasesColumns = {
+  database_name: string;
+  owner: string;
+  primary_region: string;
+  secondary_region: string;
+  regions: string[];
+  survival_goal: string;
+};
+
+export type DatabasesListResponse = { databases: string[] };
+
+export const DatabasesRequest: SqlExecutionRequest = {
```

Comment:
nit: can we change this to `databaseRequest` I don't think we have a convention of capitalizing the first letter of an object in this case, even though it's an exported object.